core/shared/src/main/scala/zio/LogLevel.scala

```diff
@@ -66,5 +66,23 @@ object LogLevel {
   val Trace: LogLevel   = LogLevel(0, "TRACE", 7)
   val None: LogLevel    = LogLevel(Int.MaxValue, "OFF", 7)
 
+  val config: Config[LogLevel] = Config.string.mapOrFail { value =>
```

Comment:
I think this should probably be in the `Config` companion object. Organizing principle would be configurations for all ZIO and standard library data types are in the `Config` companion object, and all other configurations are defined in the companion object of the data type.